zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/Collection.java

```diff
@@ -1167,16 +1167,20 @@ public boolean deleteDataVisContent(Session session, Path contentPath) throws IO
             return false;
         }
 
-        String visualisationZipUri = contentPath.toString();
+        String visualisationZipUri = resolveDataVizZipURI(contentPath);
```

Comment:
Question on naming. The function mat be for the purpose of data viz zips but in reality it's just the parent dir if the file to delete is `data.json`. Maybe something like `parentDir` or something?